frontend/src/components/GoalForm/Form.js

```diff
@@ -18,10 +17,12 @@ import AppLoadingContext from '../../AppLoadingContext';
 import './Form.scss';
 import ConditionalFields from '../ConditionalFields';
 import GoalSource from './GoalSource';
+import GoalName from './GoalName';
 
 export const BEFORE_OBJECTIVES_CREATE_GOAL = 'Enter a goal before adding an objective';
 export const BEFORE_OBJECTIVES_SELECT_RECIPIENTS = 'Select a grant number before adding an objective';
 export default function Form({
```

Comment:
This component is basically a pass-through. Here, we're conditionally displaying different goal nudge fields, so the component for the goal text is adjusted, and more props are needed and so are added to the pass-through.